### ISSUE TITLE
Outdated doc about the --chain flag

### ISSUE THREAD

**Issue by chevdor** [paritytech/substrate#5720]
The doc mentions that `--chain local` is valid but this is no longer the case.

**Comment by bkchr** [comment]
Yeah, not sure how we should solve this. The best is to just update the comment and remove spec names.

**Comment by chevdor** [comment]
We cannot be exhaustive but at least give a hint. 
I made such a proposal in #5721

**Comment by norpol** [comment]
Having checked the code on the weekend I see that there are already bunch of chains available, how about doing something along `--chain help` to print a list of available chains (and to add to the chain flags a mandatory URL to documentation of each chain?)

**Comment by chevdor** [comment]
Yes, that would be the proper option. Probably not `--chain` though but a new sub-command.
